resources/pdns_recursor_install_rhel.rb

```diff
@@ -66,3 +69,9 @@
     action :remove
   end
 end
+
+action_class.class_eval do
+  def whyrun_supported?
```

Comment:
This should have already been defined for us by Chef. The 12.5+ syntax includes this method.